event_training/models/product.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# © 2015 Grupo ESOC Ingeniería de Servicios, S.L.U. - Jairo Llopis
+
+from openerp import _, api, fields, models
+from .. import exceptions as ex
+
+
+class EventTrainingProductABC(models.AbstractModel):
+    _name = "event_training.product_abc"
```

Comment:
It comes from [Python's abc module](https://docs.python.org/2.7/library/abc.html).